### DISCUSSION TITLE
Business icon use example

### DISCUSSION THREAD

**Discussion by robyn-bautista** [twilio-labs/paste#3359] [General] [answered]
Hi! I've seen quite a few posts on business icon use:

Post 1: https://github.com/twilio-labs/paste/discussions/3085
Post 2: https://github.com/twilio-labs/paste/discussions/2146

To make sure I'm understanding paste recommendations, is the recommendation to go with [this screen](https://www.figma.com/file/8Sjunj0dodG6bJ0RnFt2XF/Channels?type=design&node-id=125%3A103355&mode=design&t=3SjDCxc1jJf6Lh1e-1) adhering to outside brand use or [this one](https://www.figma.com/file/8Sjunj0dodG6bJ0RnFt2XF/Channels?type=design&node-id=125%3A153932&mode=design&t=3SjDCxc1jJf6Lh1e-1) that adheres to our brand/product consistency?

**Comment by loreina** [comment, marked as answer]
Hi @robyn-bautista,

Paste can't add company logos into our libraries because they usually have pretty strict guidelines. We recommend using existing icons in the Paste library to convey similar sentiment and promote product consistency. If there's a use case for using other company logos, your team will have to manage those logo assets and adhere to their brand guidelines.

It's a pretty generic answer but hope it helps! If there's a specific use case you're looking into and want to share, we can provide more context-specific guidance too.
